.readthedocs.yaml

```diff
@@ -12,8 +12,13 @@ build:
     python: "3.9"
   jobs:
     pre_build:
-      - make apidoc
-      - breathe-apidoc -o docs/api apidoc/xml
+    - make apidoc
+    - breathe-apidoc -o docs/api apidoc/xml
+    post_build:
+    - |
+      if [ "$READTHEDOCS_VERSION" = "development" ]; then
```

Comment:
CI says you didn't get the YAML syntax right

Reply:
Looks like its a bug in Readthedocs's command parsing. I've pushed a workaround.